purchases/src/main/kotlin/com/revenuecat/purchases/coroutinesExtensions.kt

```diff
@@ -31,3 +31,30 @@ suspend fun Purchases.awaitCustomerInfo(
         )
     }
 }
+
+/**
+ * Fetch the configured offerings for this users. Offerings allows you to configure your in-app
+ * products vis RevenueCat and greatly simplifies management. See
+ * [the guide](https://docs.revenuecat.com/offerings) for more info.
+ *
+ * Offerings will be fetched and cached on instantiation so that, by the time they are needed,
+ * your prices are loaded for your purchase flow. Time is money.
+ *
+ * Coroutine friendly version of [Purchases.getOfferings].
+ *
+ * @warning This function is marked as [ExperimentalPreviewRevenueCatPurchasesAPI] and may change in the future.
+ * Only available in Kotlin.
+ *
+ * @throws [PurchasesException] with a [PurchasesError] if there's an error retrieving the offerings.
```

Comment:
Does Kotlin allow specifying which exception is thrown just like Java does?

Reply:
Unfortunately it doesn't: https://kotlinlang.org/docs/exceptions.html#checked-exceptions

Reply:
We can add an `@throws` annotation though. It won't specify the type but will give a small warning I think?

Reply:
I added a `@Throws(PurchasesException::class)`